.travis.yml

```diff
@@ -7,14 +7,11 @@ node_js:
   - "6"
   - "7"
 
-matrix:
-  allow_failures:
-    - node_js: "0.12"
-    - node_js: "node"
- 
 branches:
-  except:
-    - canary
+  only:
+    - master
+    - beta
+    - release
```

Comment:
Confirm, green keeper and homu branches need to be included or they won't run (likely why @nathanhammond couldn't get homu to run on a PR last night after this landed).